src/main/groovy/org/broadinstitute/orsp/IRBKeyDocuments.groovy

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.orsp
+
+enum IRBKeyDocuments {
```

Comment:
I should have read further before commenting above. I think these key documents can all be combined into one enum for clarity, something like `KeyDocument` where the constructor takes a name and an `IssueType`. You could do a lot of the filtering logic above inside the enum and take it out of the domain object.